hw_9.js

```diff
@@ -0,0 +1,71 @@
+document.addEventListener('DOMContentLoaded', function () {
+    function getRandomInt(min, max) {
+        return Math.floor(Math.random() * (max - min)) + min;
+    }
+    let limitNumber;
+    let pageNumber;
+
+    setTimeout(function () {
+        limitNumber=getRandomInt(1, 11);
+        console.log(limitNumber)
+    }, 1000);
+    setTimeout(function () {
+        pageNumber=getRandomInt(1, 11);
+        console.log(pageNumber)
+    }, 2000);
+
+    setTimeout(function () {
+        getPost(function (responseText) {
+            console.log("callback response =>", JSON.parse(responseText));
+        })
+    }, 2500);
+    setTimeout(GetPostByPromise, 2500);
+    setTimeout(GetPostByAsync, 2500);
```

Comment:
you should work with result of async function as with promise, see:

    async ensures that the function returns a promise, and wraps non-promises in it.

https://javascript.info/async-await#async-functions